rest/config_flags.go

```diff
@@ -132,7 +132,8 @@ func registerConfigFlags(config *StartupConfig, fs *flag.FlagSet) map[string]con
 		"unsupported.serverless.min_config_fetch_interval": {&config.Unsupported.Serverless.MinConfigFetchInterval, fs.String("unsupported.serverless.min_config_fetch_interval", "", "How long to cache configs fetched from the buckets for. This cache is used for requested databases that SG does not know about.")},
 		"unsupported.use_xattr_config":                     {&config.Unsupported.UseXattrConfig, fs.Bool("unsupported.use_xattr_config", false, "Store database configurations in system xattrs")},
 
-		"unsupported.user_queries": {&config.Unsupported.UserQueries, fs.Bool("unsupported.user_queries", false, "Whether user-query APIs are enabled")},
+		"unsupported.user_queries":                      {&config.Unsupported.UserQueries, fs.Bool("unsupported.user_queries", false, "Whether user-query APIs are enabled")},
+		"unsupported.allow_scopes_in_persistent_config": {&config.Unsupported.AllowScopesInPersistentConfig, fs.Bool("unsupported.allow_scopes_in_persistent_config", false, "Allows scopes in persistent config, the behavior is undefined if there are multiple databases with different collections targeting the same bucket.")},
```

Comment:
Does this flag need to be externally configured in order to support usage by our tests? (as opposed to setting a property on the serverContext inside the RestTester). It would be preferable to avoid exposing this (even as unsupported) if that's an option.

Reply:
I was just following past patterns, I've hidden this from users.